ml-agents/mlagents/trainers/trainer.py

```diff
@@ -71,6 +69,29 @@ def check_param_keys(self):
                     "brain {2}.".format(k, self.__class__, self.brain_name)
                 )
 
+    def write_tensorboard_text(self, key: str, input_dict: Dict[str, Any]) -> None:
+        """
+        Saves text to Tensorboard.
+        Note: Only works on tensorflow r1.2 or above.
+        :param key: The name of the text.
+        :param input_dict: A dictionary that will be displayed in a table on Tensorboard.
+        """
+        #        try:
```

Comment:
Remove commented out code if it won't be used.

Reply:
Good catch. This shouldn't be here at all.